chaos_test/pinger_config.yaml

```diff
@@ -57,5 +57,6 @@ applications:
       upgrade_types:
       - IN_PLACE
       - ROLLOUT
+      base_anyscale_url: https://console.anyscale-staging.com
```

Comment:
@shrekris-anyscale I was leaving the default behavior like this. Do you think it's better to just do `YOUR_BASE_ANYSCALE_URL` here and force user to set this up? Do you know if anything will upgrade automatically after this PR is merged?

Reply:
Nothing will upgrade automatically when this PR is merged. I'd prefer to pattern match the other fields and leave it as `YOUR_BASE_ANYSCALE_URL`.